bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script.sh

```diff
@@ -127,11 +129,9 @@ retry-command kubeadm join phase control-plane-prepare certs
 retry-command kubeadm join phase control-plane-prepare kubeconfig
 retry-command kubeadm join phase control-plane-prepare control-plane
 # {{ end }}
-retry-command kubeadm join phase kubelet-start
-# {{ if .ControlPlane }}
-try-or-die-command kubeadm join phase control-plane-join etcd
-retry-command kubeadm join phase control-plane-join update-status
-retry-command kubeadm join phase control-plane-join mark-control-plane
-# {{ end }}
+
+# Run kubeadm join and skip all already executed phases.
+try-or-die-command kubeadm join \
+  --skip-phases preflight,control-plane-prepare/download-certs,control-plane-prepare/certs,control-plane-prepare/kubeconfig,control-plane-prepare/control-plane,kubelet-start
```

Comment:
```suggestion
  --skip-phases preflight,control-plane-prepare,kubelet-start
```

can this be just the above, given all the sub phases were called explicitly?